rotkehlchen/tests/api/test_data_import.py

```diff
@@ -409,3 +431,196 @@ def test_data_import_custom_format(rotkehlchen_api_server, file_upload):
     assert result is True
     # And also assert data was imported succesfully
     assert_custom_cointracking(rotki)
+
+
+def test_associated_locations(database):
```

Comment:
As I wrote 2 times already. This file is the wrong place to put this test. It's a unit test about a database function. This file is for api integration tests.

You actually, after the last review, turned it into a unit test. So put it where it should be. tests/db/test_locations.py